Final_project_2/frontend/css/style.css

```diff
@@ -463,20 +463,15 @@ label  {
   margin-right: 30px;
 } 
 
-
-
 #contactform {
-  padding: 80px;
```

Comment:
If you inspected the contact form, you could see that it for some reason had no height of it's own. It looked super weird. I worked the divs around a bit to a point where the form was the height it was supposed to be and then this padding was no longer necessary.